virtual-desktop/plugin-config/webpack.base.js

```diff
@@ -108,4 +108,3 @@ module.exports = config;
 
   Copyright Contributors to the Zowe Project.
 */
-
```

Comment:
Add a comment that this is not frequently supported by Webpack and developers should move away from using this